openshift/.pipeline/Jenkinsfile

```diff
@@ -75,15 +75,12 @@ pipeline {
       StartBuildAndWait("${DYNAMICS_ADAPTER}-runtime", TOOLS_NAMESPACE)
      },
      'ICBCAdapter': {
```

Comment:
Can we name it SampleAdapter?

Reply:
Done: All ICBC names changed to Sample Adapter.